.eslintrc.json

```diff
@@ -14,8 +14,16 @@
     "curly": "warn",
     "no-var": 2,
     "prefer-const": "error",
+    // note you must disable the base rule as it can report incorrect errors
+    "semi": "off",
+    "@typescript-eslint/semi": ["error"],
     "space-before-blocks": "error",
     "spaced-comment": "warn",
+    "quotes": "off",
+    "@typescript-eslint/quotes": ["error", "single", { "avoidEscape": true }],
+    // note you must disable the base rule as it can report incorrect errors
+    "no-unused-vars": "off",
```

Comment:
That's odd, bc i swear we had this rule before, but i dont see it in the current eslint.json